src/cmdlib.sh

```diff
@@ -48,8 +48,10 @@ repository_dirs[ppc64le]=fedora-secondary
 repository_dirs[s390x]=fedora-secondary
 
 repository_dir=${repository_dirs[$arch]}
-INSTALLER=https://download.fedoraproject.org/pub/$repository_dir/releases/$release/Everything/$arch/iso/Fedora-Everything-netinst-$arch-$release-1.2.iso
-INSTALLER_CHECKSUM=https://download.fedoraproject.org/pub/$repository_dir/releases/$release/Everything/$arch/iso/Fedora-Everything-$release-1.2-$arch-CHECKSUM
+INSTALLER_MIRROR="${INSTALLER_MIRROR:-https://download.fedoraproject.org}"
+INSTALLER_URL="${INSTALLER_MIRROR}/pub/$repository_dir/releases/$release/Everything/$arch/iso"
+INSTALLER="${INSTALLER_URL}/Fedora-Everything-netinst-$arch-$release-1.2.iso"
```

Comment:
This is not a knock on this change, but this filename is not static:

F27 - https://fedora.mirror.constant.com/fedora/linux/releases/27/Everything/x86_64/iso/Fedora-Everything-netinst-x86_64-27-1.6.iso
F28 - https://fedora.mirror.constant.com/fedora/linux/releases/28/Everything/x86_64/iso/Fedora-Everything-netinst-x86_64-28-1.1.iso
F29 - https://fedora.mirror.constant.com/fedora/linux/releases/29/Everything/x86_64/iso/Fedora-Everything-netinst-x86_64-29-1.2.iso

Reply:
yeah if the intent here is to use F28 instead of F29 then ti's not going to work because f28 went out on the first beta candidate (`1.1`) where f29 went out on the 2nd beta candidate (`1.2`) so there is still a difference in the filename.

Reply:
The intent on the PR was not to change the static filenames; I would consider that a separate PR. My intent was to allow a user-specified mirror/location for the bits. 

Adding a lookup would be easy enough:
```
suffix=""
case $release in
    27) echo 'srly?'; suffix=1.6;;
    28) suffix=1.1;;
    29) suffix=1.2;;
      *) echo "unkown release"; exit 10;;
esac
```
 
But given that our stated intent is to remove the need for an installer URL, it seems unnecessary.